File: Plugins/Aspose_Barcode_Java_for_PHP/src/aspose/barcode/WorkingWithBarcodeRecognition/AdvancedBarcodeRecognitionFeatures/GetBarcodeRecognitionQuality.php
<?php
namespace Aspose\Barcode\WorkingWithBarcodeRecognition\AdvancedBarcodeRecognitionFeatures;

use com\aspose\barcoderecognition\BarCodeReadType as BarCodeReadType;
use com\aspose\barcoderecognition\BarCodeReader as BarCodeReader;

class GetBarcodeRecognitionQuality{

    public static function run($dataDir=null)
    {

        $img = $dataDir . "barcode.jpg";

        # initialize barcode reader
        $barcode_reader_type = new BarCodeReadType();
        $reader = new BarCodeReader($img, $barcode_reader_type->Code39Standard);

        # Call read method
        while (java_values($reader->read())){
            print "Barcode CodeText: " . (string)$reader->getCodeText(). " Barcode Type: ".(string)$reader->getReadType().PHP_EOL;
            $percent = $reader->getRecognitionQuality();
            print "Barcode Quality Percentage: ".(string)$percent.PHP_EOL;
        }

        # Close reader
        $reader->close();
    }
}

File: Plugins/Aspose.BarCode Java for Jython/asposebarcode/WorkingWithBarcodeRecognition/AdvancedBarcodeRecognitionFeatures/GetBarcodeRecognitionQuality.py
from asposebarcode import Settings
from com.aspose.barcoderecognition import BarCodeReadType
from com.aspose.barcoderecognition import BarCodeReader

class GetBarcodeRecognitionQuality:

    def __init__(self):
        dataDir = Settings.dataDir + 'WorkingWithBarcodeRecognition/AdvancedBarcodeRecognitionFeatures/GetBarcodeRecognitionQuality/'
        
        img = dataDir + "barcode.jpg"

        # initialize barcode reader
        barcode_reader_type = BarCodeReadType
        reader = BarCodeReader(img, barcode_reader_type.Code39Standard)

        # Call read method
        while (reader.read()):
            print "Barcode CodeText: " + reader.getCodeText()
            print " Barcode Type: "
            print reader.getReadType()
            percent = reader.getRecognitionQuality()
            print "Barcode Quality Percentage: "
            print percent
        

        # Close reader
        reader.close()


if __name__ == '__main__':        
    GetBarcodeRecognitionQuality()